addons/website_sale/static/src/xml/website_sale_utils.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+
+<!-- Products Search Bar autocomplete item -->
+<li t-name="website_sale.productsSearchBar.autocompleteItem" class="p-2">
+    <a t-att-href="item.url" class="d-block">
+        <div class="media align-items-center o_search_product_item">
+            <t t-if="options.display_image">
+                <img t-attf-src="/web/image/product.template/#{item.id}/image_small"/>
+            </t>
+            <div class="media-body px-3">
+                <t t-set="with_description" t-value="options.display_description and item.description"/>
+                <h6 t-attf-class="font-weight-bold #{with_description ? '' : 'mb-0'}" t-esc="item.name"/>
+                <t t-if="with_description">
+                    <p class="mb-0" t-esc="item.description"/>
+                </t>
+            </div>
+            <t t-if="options.display_price">
+                <b>
+                    <t t-if="item.currency_position === 'after'">
```

Comment:
can't you use a widget monetary in this case ? 

smthing ~ "widget.render_monetary_field(amount, currency)"

Reply:
This is static qweb... so yes maybe we could do something like that but that would be something new to implement